src/v0/destinations/braze/util.js

```diff
@@ -317,18 +319,51 @@ const processDeduplication = (userStore, payload, destinationId) => {
   return null;
 };
 
+function prepareGroupAndAliasBatch(arrayChunks, responseArray, destination, type) {
+
+  const headers = {
+    'Content-Type': JSON_MIME_TYPE,
+    Accept: JSON_MIME_TYPE,
+    Authorization: `Bearer ${destination.Config.restApiKey}`,
+  };
+
+  for (let chunk of arrayChunks) {
+    const response = defaultRequestConfig();
+    if (type === 'merge') {
+      response.endpoint = getAliasMergeEndPoint(getEndpointFromConfig(destination));
```

Comment:
both blocks are very similar try to write reusable function

Reply:
both block has different endpoint and payload `key` name ... other common code is separated out